.circleci/config.yml

```diff
@@ -18,7 +18,7 @@ jobs:
               exit 0
             fi
 
-            if [[ "$CIRCLE_BRANCH" == "develop" || "$CIRCLE_BRANCH" == "release/"* ]]; then
+            if [[ "$CIRCLE_BRANCH" == "develop" || "$CIRCLE_BRANCH" == "release/"* || "$CIRCLE_BRANCH" == "feat/protocol" ]]; then
```

Comment:
We should also update the `.circleci/workflows.yml` file to run the same jobs as develop.